mautic/mautic
# [UX] Create tooltips for the "Script inside Head/Footer" labels

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | [ ]
| New feature/enhancement? (use the a.x branch)      | [x]
| Deprecations?                          | [ ]
| BC breaks? (use the c.x branch)        | [ ]
| Automated tests included?              | [ ] 
| Related user documentation PR URL      | mautic/mautic-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation#... 
| Issue(s) addressed                     | Fixes #... 



#### Description:

When we are creating a page, there is the option for you to place a script but there is no specification of how this script should be placed. If you place it without HTML tags, the code is printed on the page screen, this shouldn't happen.



#### Steps to test this PR:


1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2. Open Landing pages
3. Create a new
4. See the new tooltips on the right side of the screen

![image](https://github.com/mautic/mautic/assets/116097999/2f6cdf6c-043b-4c7d-ba21-041a997a83de)
